frontend/src/app/features/bim/bcf/bcf-wp-attribute-group/bcf-wp-attribute-group.component.ts

```diff
@@ -163,7 +163,7 @@ export class BcfWpAttributeGroupComponent extends UntilDestroyedMixin implements
   }
 
   async initialize(workPackage:WorkPackageResource) {
-    this.projectId = idFromLink(workPackage.project.href);
+    this.projectId = String(idFromLink(workPackage.project.href));
```

Comment:
I've updated `idFromLink` to return a number, since the API ids are numbers (I think we should strive to align our models with those coming from the backend) however, that means I had to go everywhere it's used and wants a string.

Reply:
I reverted this changes because it broke huge parts of the application (e.g. the WP page). Since it is not a necessary part of this PR, I argue not to do it here.